README.md

```diff
@@ -68,13 +79,5 @@ To start OGV run
 ```
 App will be running at http://localhost:3000
 
-
-## BRL-CAD
-* Downloading
-	* Go to https://brlcad.org/d/download
-	* Download file for your preferred system
-* Installing
-```
-sudo dpkg -i brl*.deb
-```
-Note: Install all the packages in user permission.
+##Note
+If you are facing any permission issues then do not use sudo while installing the packages.
```

Comment:
Why are you making this recommendation?  It's potentially flawed, potentially correct.

Reply:
Actually, when i was installing the meteor packages I by mistake used 'sudo' and faced some permission errors. I will add more specific details with new commits.